box_it_script/index.js

```diff
@@ -0,0 +1,259 @@
+
+/** 
+ * Author: Kevin Njunge
+ * Assignment: Box-It-Script
+ * Due Date: March 15, 2019
+ * 
+ * 
+ * 
+ * To execute:
+ * 1.) cd box_it_script/
+ * 2.) run chmod +x index.js to make it executable with ./
+ * 3.) to execute the file, either 
+ *     a.) run ./index.js <filename> with either .csv or .txt extension to read from file. Or
+ *   OR
+ *     b.) run ./index.js <string 1> <string 2> ... to read from terminal.
+ * 
+ * ie.) 
+ * 
+ * ./index.js
+ * ┌┐
+ * └┘
+ * 
+ * ./index.js empty.csv
+ * ┌┐
+ * └┘
+ * 
+ * ./index.js 'abc,defg' hijkl,sfsaf abc def
+ * ┌─────┬─────┐
+ * |abc  |defg |
+ * |─────┼─────|
+ * |hijkl|sfsaf|
+ * |─────┼─────|
+ * |abc  |     |
+ * |─────┼─────|
+ * |def  |     |
+ * └─────┴─────┘
+ * 
+ * ./index.js 'Jon Snow' 'Daenerys Targaryen' 'Arya Stark'
+ * ┌──────────────────┐
+*  |Jon Snow          |
+*  |──────────────────|
+*  |Daenerys Targaryen|
+*  |──────────────────|
+*  |Arya Stark        |
+*  └──────────────────┘
+* 
+* 
+* ./index.js twocolumns.csv
+* 
+* 
+* ┌──────────────────┬─────────┐
+* |Names             |House    |
+* |──────────────────┼─────────|
+* |Jon Snow          |Stark    |
+* |──────────────────┼─────────|
+* |Daenerys Targaryen|Targaryen|
+* └──────────────────┴─────────┘
+*
+*
+*
+* ./index.js onecolumn.txt
+* ┌──────────────────┐
+* |Names             |
+* |──────────────────|
+* |House             |
+* |──────────────────|
+* |Jon Snow          |
+* |──────────────────|
+* |Stark             |
+* |──────────────────|
+* |Daenerys Targaryen|
+* |──────────────────|
+* |Targaryen         |
+* └──────────────────┘
+* 
+* **/
+
+// #! /usr/bin/env node -> runs env command which finds where 
+// node.js is installed and runs it.
+// inorder to make file: index.js executable with ./ you'll first need
+// to run chmod +x index.js
+// https://jrgraphix.net/r/Unicode/2500-257F
+
+
+const args = process.argv.slice(2)
+
+// schematic of box
+const box ={
+    topLeftCorner: '\u250C',
+    topRightCorner: '\u2510',
+    bottomLeftCorner: '\u2514',
+    bottomRightCorner: '\u2518',
+    middleTopCorner: '\u252c',
+    middleLine:'\u253c',
+    middleBottomCorner:'\u2534',
+    horizantalLine: '\u2500',
+    verticalLine:'|',
+    space: ' ',
+    newLine:'\n'
+}
+/**
+ * @param  {int} n
+ * drawLine: draws line of length n
+ */
+function drawLine(n){
+    let line = ''
+    for(let i = 0; i<n; i++){
+        line += box.horizantalLine
+    }
+    return line
+}
+// To do
+/**
+ * @param  {int} n
+ * @param  {string} beginning='\u250C'
+ * @param  {string} ending='\u2510'
+ * drawTopBorder: draws top border of length n
+ */
+function drawTopBorder(n, beginning= box.topLeftCorner, ending=box.topRightCorner){
+    return `${beginning}${drawLine(n)}${ending}`
+}
+/**
+ * @param  {int} n
+ * @param  {string} beginning = '|'
+ * @param  {string} ending='|'
+ * drawMiddleBorder: draws middle border of length n
+ */
+function drawMiddleBorder(n,beginning=box.verticalLine,ending=box.verticalLine){
```

Comment:
make sure to add spaces between commas! Might be a good idea to look into a linter (eslint) for javascript